cmake/toolchain/zephyr/target.cmake

```diff
@@ -15,4 +15,10 @@ else()
     set(SYSROOT_DIR ${TOOLCHAIN_HOME}/xtensa/${SOC_TOOLCHAIN_NAME}/${SYSROOT_TARGET})
     set(CROSS_COMPILE ${TOOLCHAIN_HOME}/xtensa/${SOC_TOOLCHAIN_NAME}/${CROSS_COMPILE_TARGET}/bin/${CROSS_COMPILE_TARGET}-)
   endif()
+
+endif()
+
+# The ARM/ARM64 split requires at least the 0.12.4 SDK
+if(("${ARCH}" STREQUAL "arm64") AND (${SDK_VERSION} VERSION_LESS_EQUAL 0.12.3))
```

Comment:
actually this should go here:
https://github.com/zephyrproject-rtos/zephyr/blob/55771d36a7a7a452b962c81bdef832fcb7be570b/cmake/verify-toolchain.cmake#L18-L19

and then take the form:
```
if("${ARCH}" STREQUAL "arm64")
  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.12.4)
else()
  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.11.3)
endif()
```

and then we should update:
https://github.com/zephyrproject-rtos/zephyr/blob/55771d36a7a7a452b962c81bdef832fcb7be570b/cmake/verify-toolchain.cmake#L70
and 
https://github.com/zephyrproject-rtos/zephyr/blob/55771d36a7a7a452b962c81bdef832fcb7be570b/cmake/verify-toolchain.cmake#L75
to include `REQUIRED`.

That will give regular CMake error in the form of:
```
CMake Error at /projects/github/ncs/zephyr/cmake/verify-toolchain.cmake:75 (find_package):
  Could not find a configuration file for package "Zephyr-sdk" that is
  compatible with requested version "0.12.4".

  The following configuration files were considered but not accepted:

    /opt/zephyr-sdk-0.12.3/cmake/Zephyr-sdkConfig.cmake, version: 0.12.3
```

Only thing to decide is whether it's time to remove support for `0.11.1` and `0.11.2`.
When adding `REQUIRED`, then it will no longer be possible to point to a `0.11.1` or `0.11.2`.
@galak thoughts ?

Reply:
@galak ping, so that I can push a new revision

Reply:
@abrodkin @evgeniy-paltsev could you comment.

Reply:
Discussed at Toolchain WG.
Will update to minimum 0.12.